Automattic/woocommerce-payments
# Fraud & Risk tools improvements from feedback

Fixes Automattic/woocommerce-payments-server#3181

#### Changes proposed in this Pull Request

Layout changes:
- Adjust advanced fraud settings container width
- Adjust advanced fraud rule cards spacings
- Adjust discoverability banner spacings

Improvements:
- Add "Held for review and manually canceled" fraud meta box
- Add "Back to Payments Settings" and "Save changes" buttons to the advanced settings footer
- Refetch "Risk review" data after approving or canceling an on-hold order









#### Testing instructions





**Pre-requisites**
* Checkout to this branch `fix/3181-demo-feedback`
* Update the version on `./woocommerce-payments.php` to be at least `5.7.0`
* Enable the fraud protection feature flag
  * If this is the first time, run `npm run wp option add wcpay_fraud_protection_settings_active 1`
  * If you already have it set, run `npm run wp option update wcpay_fraud_protection_settings_active 1`
* Build the client `npm run start`
* Listen to Stripe webhooks `npm run listen` – or equivalent

**Testing**
* Go to **Payments > Settings** and scroll to the **Fraud protection** section.
* Select the **Advanced** protection level and click on "Configure"
* Scroll to the bottom of the page
  * You should see the "Back to Payments Settings" and "Save changes" buttons
* Scroll to the **Purchase Price Threshold** rule
* Enable the filtering by clicking on "Screen transactions for abnormal purchase prices"
* Set **1000** to the "Minimum purchase price" input
* Disable the "Block Payment" check (If it was checked)
* Save the changes
* Open an incognito browser instance, go to your shop, and try to buy something – it should succeed
* Open **WooCommerce > Orders** and click on the first item – it should be the order that you've just created
* Click on "Review payment"

<img width="290" alt="image" src="https://user-images.githubusercontent.com/16882226/228725287-970e76f3-2807-4ad8-a372-a3c22c2bf5a6.png">

* Click on "Block transaction"
* Go back to the order details page
* You should see the new fraud meta box:

<img width="290" alt="image" src="https://user-images.githubusercontent.com/16882226/228725488-c4b81b29-aefb-460b-aea3-b8e3d5a5f221.png">

-------------------


**Post merge**
